salt/reboot/init.sls

```diff
@@ -2,6 +2,9 @@
 # Configuration for the reboot manager
 ##################################################
 
+include:
+  - etcd-proxy
```

Comment:
I thought we had decided not to include `etc-proxy` from the reboot setup stage....